Update index.md (#99)
trywilco/wilco-docs: docs/building-your-quest/actions/ready-response-message.md

---
title: bot_message
parent: Actions
grand_parent: Building Your Quest
nav_order: 2
---

# bot_message

## Description

**Type**: Action

Sends a message to the user on behalf of one of the bots. 

## Params

- **person:** The name of the bot. e.g., `keen` or `lucca`.    

## Outputs

No additional info is added to the global payload outputs.

## Usage Example

```yaml
startFlow:
  do:
  - actionId: ready_message
    params:
      person: keen
```

> The action `ready_message` is used to send "ready"/"not ready" button when you want to get the readiness confirmation from the usre.

Reviewer: ```suggestion
The action `ready_message` sends "ready"/"not ready" buttons to the user in order to confirm their readiness.
```

> Here is how it will look like:

Reviewer: ```suggestion
Here is what it will look like:
```
